frontend/src/components/RecordsList.tsx

```diff
@@ -10,20 +10,27 @@ import styles from "./records.module.scss";
 import "ag-grid-community/styles/ag-grid.css";
 import "ag-grid-community/styles/ag-theme-alpine.css";
 import "ag-grid-enterprise";
-import { ColDef, IServerSideGetRowsParams } from "ag-grid-community";
+import {
+  ColDef,
+  IServerSideDatasource,
+  IServerSideGetRowsParams,
+} from "ag-grid-community";
 import { DataName, useHookLazyGeneric } from "../shared/types";
 import SamplesList from "./SamplesList";
 import { Sample, SampleWhere } from "../generated/graphql";
-import { defaultColDef } from "../shared/helpers";
+import {
+  defaultColDef,
+  prepareSampleMetadataForAgGrid,
+} from "../shared/helpers";
 import { PatientIdsTriplet } from "../pages/patients/PatientsPage";
 import { ErrorMessage, LoadingSpinner, Toolbar } from "../shared/tableElements";
 
 interface IRecordsListProps {
   colDefs: ColDef[];
   dataName: DataName;
-  nodeName?: string;
```

Comment:
This prop was here basically to handle the "Patients" page querying data by a node of a different name ("PatientAlias"), unlike the "Requests" and "Cohorts" pages. Now that the "Patients" page queries data of node "Patients", we no longer need this prop.